ontologies/Sierra_film.json

```diff
@@ -0,0 +1,180 @@
+{
+  "@context": "https://api.wellcomecollection.org/catalogue/v0/context.json",
+  "id": "8rm28aw",
+  "type": "Work",
+  "identifiers": [
+    {
+      "label": "b24993207",
+      "authority": "Sierra"
+    }
+  ],
+  "label": "Are health tests a good idea? An Horizon special",
+  "creators": [
+    {
+      "label": "Davies, Serena, television producer, television director",
```

Comment:
Yes, absolutely: it's something that the new cataloguing paradigms for books (RDA) and archives (RiC) are both keen on, and I keep banging on to Axiell that future versions of CALM are going to have to support this.  Even if the data currently coming down the pipe from CALM doesn't show this information, I'm hopeful that it will.  So yes, Agents ontology will need to include this!